### PR TITLE
Update documentation

### LINKED ISSUE #2: Update Documentation
- Update User Guides
- Update Developer Guides
- Update README and AboutMe documents

### PR DESCRIPTION
**Pull request by AhQuanz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -126,8 +122,16 @@ Format: `exit`
...
+**addcheese** | `addcheese t/CHEESE_TYPE q/QUANTITY`<br> Example: `addcheese t/Parmesan q/2`
+**addcustomer** | `addcustomer n/CUSTOMER_NAME p/PHONE_NUMBER a/ADDRESS` <br> Example: `addcustomer n/John Doe p/65555555 a/Blk 436 Serangoon Gardens St 26 #01-01`
+**removeorder** | `removeorder ORDER_INDEX` <br> Example: `removeorder 2`
+**removecheese** | `removecheese CHEESE_INDEX` <br> Example: `removecheese 5`
+**removecustomer** | `removecustomer p/PHONE_NUMBER` <br> Example: `removecustomer p/65555555`
+**listorder** | `listorder`
+**listcheese** | `listcheese t/CHEESE_TYPE` <br> Example: `listcheese t/Mozzarella`
+**listcustomer** | `listcustomer`
+**done** | `done INDEX` <br> Example: `done 2`
+**search** | `search n/Name` <br> Example: `search n/Betty`
```
Comment (daniellau88) [inline docs/UserGuide.md]: For this portion, I think would be better to use the one in @weixue123 's branch, so might need to merge it in first.

[refer to this portion](https://github.com/AY2021S2-CS2103-W16-2/tp/compare/master...weixue123:inc/doc#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR146)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -105,6 +105,45 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**MSS**
+
+1. User enters the index of cheese to be removed.
+2. CHIM removes the cheese from the list of cheese.
+   
+   Use case ends.
+
+**Extensions**
+* 1a. The list is empty.
+    * 1a1.CHIM responds that there are no existing customers.
```
Comment (daniellau88) [inline docs/DeveloperGuide.md]: Would be nice to have spacing between CHIM and the full stop

File: docs/UserGuide.md [hunk 3]
```diff
@@ -18,30 +21,6 @@ Chesse Inventory Management (CHIM) is a **desktop app for managing contacts, opt
 
 ## Features
 
-<div markdown="block" class="alert alert-info">
-
-**:information_source: Notes about the command format:**<br>
```
Comment (daniellau88) [inline docs/UserGuide.md]: Are we keeping this portion? Can refer below for the modified command in @weixue123 's repo

[code portion](https://github.com/AY2021S2-CS2103-W16-2/tp/compare/master...weixue123:inc/doc#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR28)

File: docs/UserGuide.md (hunk 1 above)
Comment (AhQuanz) [reply docs/UserGuide.md]: Sure. Merge his PR and then i will resolve the conflict and add in the missing actions

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (AhQuanz) [reply docs/DeveloperGuide.md]: Noted.

File: docs/UserGuide.md (hunk 3 above)
Comment (AhQuanz) [reply docs/UserGuide.md]: Yeap , will merge his changes into my branch after his PR has been merged

**Comment by aaronsms** [comment]
I will recommend squashing your commits before merging if possible.

**Review by aaronsms — APPROVED** [review]
Okay looks good.

**Pull request merged**